src/router.ts

```diff
@@ -39,6 +42,9 @@ const routes = [
   { path: '/study-tools/commentaries', component: Commentaries }
 ]
 
+if (Config.environment === 'dev')
+  routes.push({ path: '/test', component: ComponentTestPage })
```

Comment:
When the new env variable is set to dev it will add this test route so we can test things without necessarily needing a page to display it on